src/main/java/com/xjeffrose/chicago/ChicagoObjectEncoder.java

```diff
@@ -76,10 +76,8 @@ protected void encode(ChannelHandlerContext ctx, Object msg, List<Object> out) t
     if (msg instanceof ChicagoMessage) {
       out.add(encode(ctx, (ChicagoMessage) msg));
     } else if (msg instanceof ByteBuf) {
-      //ByteBuf bb = ctx.alloc().directBuffer();
-      //bb.writeBytes(((ByteBuf)msg).readBytes(((ByteBuf) msg).readableBytes()));
-      //out.add(bb);
-      out.add(((ByteBuf) msg).retain());
```

Comment:
Wasn't this retain needed to compensate for the release in the parent Encoder? 

https://github.com/netty/netty/blob/00deb2efd560cb9b2d85ccdefbcfa5b13ab1b463/codec/src/main/java/io/netty/handler/codec/MessageToMessageEncoder.java#L90